.env

```diff
@@ -0,0 +1,8 @@
+# TODO: change ID and COMPONENTS definition because of refactoring of service names
+REACT_APP_CONTEXT=development
+REACT_APP_ID=al-ontographer
+REACT_APP_URL=http://localhost:3000
+REACT_APP_COMPONENTS=YWwtc2dvdi1zZXJ2ZXI6CiAgbmFtZTogU0dvViBTZXJ2aWNlCiAgdXJsOiBodHRwOi8vbG9jYWxob3N0L21vZGVsdWplbWUvc2x1emJ5L3Nnb3Ytc2VydmVyCiAgbWV0YToge30KYWwtZGItc2VydmVyOgogIG5hbWU6IEEgZ3JhcGggZGF0YWJhc2UKICB1cmw6IGh0dHA6Ly9sb2NhbGhvc3Q6NzIwMC9yZXBvc2l0b3JpZXMvdGVzdE9HCiAgbWV0YToge30KYWwtYXV0aC1zZXJ2ZXI6CiAgbmFtZTogT0lEQyBBdXRoZW50aWNhdGlvbiBTZXJ2aWNlCiAgdXJsOiBodHRwOi8vbG9jYWxob3N0OjgwODAvcmVhbG1zL2Fzc2VtYmx5LWxpbmUKICBtZXRhOiB7fQphbC1vbnRvZ3JhcGhlcjoKICBuYW1lOiBvbnRvR3JhcGhlcgogIHVybDogaHR0cDovL2xvY2FsaG9zdC9tb2RlbHVqZW1lL3YtbsOhc3Ryb2ppL29udG9ncmFwaGVyCiAgbWV0YToKICAgIHdvcmtzcGFjZS1wYXRoOiAnLz93b3Jrc3BhY2U9JVdPUktTUEFDRV9JUkklJwphbC10ZXJtaXQtc2VydmVyOgogIG5hbWU6IFRlcm1JVCBiYWNrZW5kCiAgdXJsOiBodHRwOi8vbG9jYWxob3N0L21vZGVsdWplbWUvc2x1emJ5L3Rlcm1pdC1zZXJ2ZXIKICBtZXRhOiB7fQphbC10ZXJtaXQ6CiAgbmFtZTogVGVybUlUCiAgdXJsOiBodHRwOi8vbG9jYWxob3N0L21vZGVsdWplbWUvdi1uw6FzdHJvamkvdGVybWl0CiAgbWV0YToKICAgIHdvcmtzcGFjZS1wYXRoOiAnLyN2b2NhYnVsYXJpZXM/d29ya3NwYWNlPSVXT1JLU1BBQ0VfSVJJJScKYWwtbWlzc2lvbi1jb250cm9sOgogIG5hbWU6IE1pc3Npb24gQ29udHJvbAogIHVybDogaHR0cDovL2xvY2FsaG9zdC9tb2RlbHVqZW1lCiAgbWV0YToge30KYWwtaXNzdWUtdHJhY2tlcjoKICBuYW1lOiBJc3N1ZSBUcmFja2VyCiAgdXJsOiBodHRwczovL2dpdGh1Yi5jb20vb3BlbmRhdGEtbXZjci9zZ292LWFzc2VtYmx5LWxpbmUvaXNzdWVzCiAgbWV0YToKICAgIG5ldy1idWc6IGh0dHBzOi8vZ2l0aHViLmNvbS9vcGVuZGF0YS1tdmNyL3Nnb3YtYXNzZW1ibHktbGluZS9pc3N1ZXMvbmV3P2xhYmVscz1idWcmdGVtcGxhdGU9cG8tYWRhdmVrLW5hLW9wcmF2dS5tZCZ0aXRsZT0KICAgIG5ldy1mZWF0dXJlOiBodHRwczovL2dpdGh1Yi5jb20vb3BlbmRhdGEtbXZjci9zZ292LWFzc2VtYmx5LWxpbmUvaXNzdWVzL25ldz9sYWJlbHM9ZW5oYW5jZW1lbnQmdGVtcGxhdGU9cG8tYWRhdmVrLW5hLW5vdm91LWZ1bmtjaW9uYWxpdHUubWQmdGl0bGU9Cg==
+BROWSER=none
```

Comment:
It looks bit odd having all env variables having certain prefix and the BROWSER one doesn't